tests_aws/test_registry/test_data_sync.py

```diff
@@ -26,222 +78,251 @@ def s3_cloudinterface():
     return s3_cloudinterface
 
 
-def test_data_sync_project_id(s3_cloudinterface):
-
-    try:
-        s3_cloudinterface.sync_push(
-            local_path=local_registry, remote_path=TEST_REMOTE_REGISTRY, exclude=SYNC_EXCLUDE_LIST
-        )
-        registry_manager = RegistryManager.load(
-            local_registry_data_sync,
-            remote_registry,
-            offline_mode=False,
-            dry_run_mode=False,
-            no_prompts=True,
-        )
-
-        project_id = "efs_2018"
-        dataset_id = None
-        registry_manager.data_sync(project_id, dataset_id)
-
-        assert (
-            len(
-                LocalFilesystem().listdir(local_registry_data_sync / "data", directories_only=True)
+def test_data_sync_project_id(s3_cloudinterface, local_registry):
```

Comment:
Is it necessary for both the test and the `s3_cloudinterface` fixture to run the `local_registry` fixture?

Reply:
I can do that, I just didn't want to repeat that code for every function.